src/common/world.c

```diff
@@ -105,6 +105,12 @@ void get_world_cmd() {
 					get_broadcast();
 					is_showing_broadcast = true;
 					break;
+				case 6: // enable info
```

Comment:
I don't really understand what is happening here.  Are these commands that can be accessed via an API GET?  What do the numbers mean?  I guessed.

Reply:
yes, these are the codes that I would need to add to the server to send the client a command that it should turn on/off its info
However, it means I need to implement it in the server too now to work!

Reply:
Where is the server code?  I can take a look at adding that functionality.

Reply:
Assuming this? https://github.com/markjfisher/bounce-world

Reply:
As noted on Discord - looked into this but it's a bit over my head currently.  Is this a blocker to merging this change?  If so would it help to just remove the changes to `world.c` for now & come back to exposing the function as an API endpoint later?